lib/floki.ex

```diff
@@ -1,4 +1,5 @@
 defmodule Floki do
+  require Floki
```

Comment:
This is not needed :)
```suggestion
```

Reply:
Sorry, some sort of brain fart.

Reply:
VS Code added that line accidentally. I removed it. Good catch.